Can't resolve 'react-p5'

Hey! I have a problem:
With the following code I get the error 'Module not found: Can't resolve 'react-p5' in 'C:\....'. 
I ran 'npm i p5 react-p5' and also checked the package.json. If I hover over the importet 'Sketch' it even shows the proper information on that. Everything seems to be correct but it just won't work and I couldn't find anything in the Internet.

```
import React, {useState,useRef} from 'react'
import Sketch from 'react-p5'

const Seperator = () => {
   const sketchContainer = useRef(null)
   let currentPage
   const preload = p5 => {
      p5.loadImage('../resources/test.png', img => {
         currentPage = img
      })
   }
   const setup = (p5, sketchContainer) => {
      p5.createCanvas(500, 500, p5.WEBGL).parent(sketchContainer)
   }
   const draw = p5 => {
      p5.image(currentPage, 0, 0)
   }

   return (
      <div ref={sketchContainer}>
         <Sketch preload={preload} setup={setup} draw={draw} />
      </div>
   )
}
export default Seperator
```

Hello,
I'm also facing a similar issue in a TypeScript project.
After creating a separate component and exporting it:
```typescript
import React from "react";
import Sketch from "react-p5"
import p5Types from "p5"; //Import this for typechecking and intellisense

interface MainSketchProps {
	//Your component props
}

export const MainSketch: React.FC<MainSketchProps> = (props: MainSketchProps) => {
	let x = 50;
	const y = 50;

	//See annotations in JS for more information
	const setup = (p5: p5Types, canvasParentRef : Element) => {
		p5.createCanvas(500, 500).parent(canvasParentRef);
	};

	const draw = (p5: p5Types) => {
		p5.background(0);
		p5.ellipse(x, y, 70, 70);
		x++;
	};

	return <Sketch setup={setup} draw={draw} />;
};
```
Then importing it:
```typescript
import React from 'react';
import './App.css';
import {MainSketch} from './Components/MainSketch';

function App() {
  return (
    <div className="App">
      <MainSketch/>
    </div>
  );
}

export default App;
```
I get this after running `npm start`, I get:
```bash
Module not found: Can't resolve 'react-p5' in 'pathToProject\project-name\src\Components'
```
I created the project using `npx create-react-app --template typescript`

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -25,6 +25,9 @@ jobs:
       - name: Install dependencies
         run: |
           npm i
+      - name: Build
+        run: |
+          npm run build
       - name: Bump package version
         run: |
           npm version patch -m "chore: Upgrade to %s"

